src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -0,0 +1,121 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalPersons.ALICE;
+import static seedu.address.testutil.TypicalPersons.BENSON;
+import static seedu.address.testutil.TypicalPersons.CARL;
+import static seedu.address.testutil.TypicalPersons.DANIEL;
+import static seedu.address.testutil.TypicalPersons.ELLE;
+import static seedu.address.testutil.TypicalPersons.FIONA;
+import static seedu.address.testutil.TypicalPersons.GEORGE;
+import static seedu.address.testutil.TypicalPersons.getShuffledTypicalAddressBook;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+
+public class SortCommandTest {
+    private Model getShuffledModel() {
+        return new ModelManager(getShuffledTypicalAddressBook(), new UserPrefs());
+    }
+
+    @Test
+    public void execute_sortByNameAscending_success() {
+        Model model = getShuffledModel();
+        SortCommand command = SortCommand.sortByName(1);
+        CommandResult result = command.execute(model);
+        String expectedMessage = "Sorted by Name in Ascending order.";
+
+        List<Person> sortedList = model.getFilteredPersonList();
+        List<Person> expectedList = List.of(ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE);
+
+        assertEquals(expectedMessage, result.getFeedbackToUser());
+        assertEquals(expectedList, sortedList);
+    }
+
+    @Test
+    public void execute_sortByNameDescending_success() {
+        Model model = getShuffledModel();
+        SortCommand command = SortCommand.sortByName(-1);
```

Comment:
Magic number here as well. Perhaps can have constant variables ASCENDING and DESCENDING to depict the integer values 1 and -1 at the start?

Reply:
I have adopted this. Thanks for the suggestion!